src/Components/Publishing/Constants.ts

```diff
@@ -79,17 +84,25 @@ export const getDate = (date, format: DateFormat = "default") => {
         .tz("America/New_York")
         .format("MMM D, YYYY")
     case "verbose":
-      const today = moment()
-      const day = today.isSame(moment(date), "day")
+      const day = isToday
         ? "Today"
         : moment(date)
             .tz("America/New_York")
             .format("MMM D, YYYY")
-
       const time = moment(date)
         .tz("America/New_York")
         .format("h:mm a")
       return `${day} at ${time}`
+    case "news":
+      return isToday
```

Comment:
"news" is maybe not the best in the context of the other types here, open to suggestions -- nothing stood out immediately.